packages/backend/src/server/web/views/base.pug

```diff
@@ -21,6 +21,8 @@ html
 		meta(name='referrer' content='origin')
 		meta(name='theme-color' content='#86b300')
 		meta(name='theme-color-orig' content='#86b300')
+		meta(property='twitter:card' content='summary')
+		meta(property='twitter:title' content= instanceName || 'Misskey')
```

Comment:
```suggestion
```

As explained, It will be better if we use OpenGraph prefix meta tag instead of twitter prefix, 
And also, since this is a base template, we don't have to declare their title.